Runtime/Common/GuidHelper.cs

```diff
@@ -17,5 +17,11 @@ public static Guid FromLong(long source)
             Array.Copy(BitConverter.GetBytes(source), guidData, 8);
             return new Guid(guidData);
         }
+
+        public static bool IsEmptyGuid(string guid)
+        {
+            const string emptyGuid = "00000000-0000-0000-0000-000000000000";
+            return string.IsNullOrEmpty(guid) || guid == emptyGuid;
```

Comment:
Should you use .equals for comparison in c#?
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/equality-operators#string-equality

Ok, the answer is no, never mind :)

Reply:
`== ` == `Equals